CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 1.2.4 (2018-03-14)
+
+### Fixes
+
+- Error responses are now user errors by default, rather than internal errors [weaveworks/flux#942](https://github.com/weaveworks/flux/pull/942)
```

Comment:
"application errors" rather than "user errors" (granted the former is how it's labelled in code; a mistake).

Reply:
We went one better and just omitted this change altogether, for the record. This change is unlikely to be visible to our users.